tests/libtelepathy/simple-account-manager.c

```diff
@@ -11,6 +11,7 @@
 
 #include "simple-account-manager.h"
 
+#include <telepathy-glib/telepathy-glib.h>
```

Comment:
The telepathy-glib.h would be the generic header which includes nearly everything below. So either could then delete almost all below, or alternatively if something is missing now it could be some more specific include instead of this? The commit message wasn't clear on what was missing.

Applies to the other file too.